storage/container/dictionary.h

```diff
@@ -52,6 +52,7 @@ class GenericDictionary {
 
   bool Empty() const { return map_.empty(); }
   size_t Size() const { return map_.size(); }
+  bool Has(sfinae::CF<key_t> x) const { return map_.find(x) != map_.end(); }
```

Comment:
Add the same method for matrix containers as well?

Reply:
:+1: ed9a57fe59666802824c3e4fbb05b6831e28127e, 93803336f6c2ae57af28adf51b019cce25213a4b